microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -435,14 +452,20 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             self.dpdk_path = self.node.get_pure_path(work_path).joinpath(
                 self._dpdk_repo_path_name
             )
-        self.find_testpmd_binary(assert_on_fail=False)
+        self._determine_network_hardware()
+        # if dpdk is already installed, find the binary and check the version
+        if self.find_testpmd_binary(assert_on_fail=False):
+            pkgconfig = self.node.tools[Pkgconfig]
+            if pkgconfig.get_package_info("libdpdk", check_exists=True):
```

Comment:
This method is confusing. If it's to check a package exists or not, it needs a method like `exists_package`

Reply:
added package_info_exists